src/GraphQL/Resolver.hs

```diff
@@ -41,6 +41,8 @@ import GraphQL.API
   )
 import qualified GraphQL.API as API
 import qualified GraphQL.Value as GValue
+import GraphQL.Value.FromValue (FromValue(..))
+import GraphQL.Value.ToValue (ToValue(..))
 import GraphQL.Value (Value)
```

Comment:
I don't think so, because that introduces a circular dependency (`FromValue` imports `Value`).